Minio/RequestExtensions.cs

```diff
@@ -145,9 +145,13 @@ await requestMessageBuilder.ResponseWriter(responseResult.ContentStream, cancell
                 throw new BucketNotFoundException();
 
             if (responseResult is not null)
+            {
                 responseResult.Exception = ex;
-            else
-                responseResult = new ResponseResult(request, ex);
+                throw;
```

Comment:
It makes no sense to set `responseResult.Exception` in the previous line and now rethrow the exception. The caller of this function will never receive the `responseResult`.

Reply:
removed!